perpVects/QFPHelpers.h

```diff
@@ -92,6 +97,30 @@ namespace FPHelpers {
     return val & (~zero >> 48);
   }
 
+  // template <typename T>
+  // int
+  // swap_float_int(T val){
+  //   return *reinterpret_cast<int*>(&val);
+  // }
+
+  auto
+  swap_float_int(unsigned int val) -> float;
```

Comment:
The unnecessary auto usage here I already commented on, but reiterate that it makes it harder to read.